cmd/kubeadm/app/phases/kubeconfig/kubeconfig.go

```diff
@@ -694,9 +694,7 @@ func EnsureAdminClusterRoleBindingImpl(ctx context.Context, adminClient, superAd
 			); err != nil {
 				lastError = err
 				if apierrors.IsAlreadyExists(err) {
-					// This should not happen, as the previous "create" call that uses
-					// the admin.conf should have passed. Return the error.
-					return true, err
+					return true, nil
```

Comment:
If admin client returns `Forbidden` and super admin client returns `already exists`, should we return err?

Reply:
/cc @danwinship @neolit123 

After https://github.com/kubernetes/kubernetes/pull/122906 is merged, the kubeadm behavior will not change and the CI should be fixed. 

I still think this is not correct here.

Reply:
This seems right. But would you mind adding some log to trace the behaviors?

Reply:
Add a simplest log here.

Reply:
/lgtm
thanks